validator_client/validator_store/src/lib.rs

```diff
@@ -77,6 +95,12 @@ pub struct ValidatorStore<T, E: EthSpec> {
     _phantom: PhantomData<E>,
 }
 
+impl<T: SlotClock + 'static, E: EthSpec> DoppelGangerValidatorStore for ValidatorStore<T, E> {
```

Comment:
I think we should rename this trait. Doppelganger with a capital G looks really weird

```suggestion
impl<T: SlotClock + 'static, E: EthSpec> DoppelgangerValidatorStore for ValidatorStore<T, E> {
```